typedb/api/concept/value/value.py

```diff
@@ -0,0 +1,124 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+from abc import ABC, abstractmethod
+from datetime import datetime
+from typing import TYPE_CHECKING, Mapping, Union
+
+from typedb.api.concept.concept import Concept, ValueType
+from typedb.api.connection.transaction import TypeDBTransaction
+from typedb.common.exception import TypeDBClientException, VALUE_HAS_NO_REMOTE
+
+if TYPE_CHECKING:
+    pass
```

Comment:
We should just delete this, right? It's a no-op
```suggestion
```

Reply:
Sure